set.mm

```diff
@@ -50761,14 +50761,13 @@ We have not yet defined relations ( ~ df-rel ), but here we introduce a few
       PVHBWAVOVGVNVFVEFUKULUMUNUOUPURUQUSUTVA $.
   $}
 
-  $( Irreflexivity of the epsilon relation: a class founded by epsilon is not a
-     member of itself.  (Contributed by NM, 18-Apr-1994.)  (Revised by Mario
-     Carneiro, 22-Jun-2015.) $)
+  $( A well-founded class does not belong to itself.  (Contributed by NM,
+     18-Apr-1994.)  (Revised by Mario Carneiro, 22-Jun-2015.) $)
```

Comment:
To understand this comment, the term "well-founded class" must be defined/explained. In my understanding, a class is "well-founded" if the epsilon relation is well-founded over this class (`Fr` is primarily a predicate for a relation ("well-formed relation")). By this , we would get a correlation with the `_E` in the theorem.